EnglishFilter/sections/specific.txt

```diff
@@ -413,6 +413,7 @@ programmingoneonone.com##.close-fixedSd
 programmingoneonone.com###Image1
 programmingoneonone.com###Image2
 opentuition.com###custom_html-72
+||videoszoofilia.net/*/banner
```

Comment:
Actually, it's a Spanish website.
Please add this rule to the `AdGuard Spanish/Portuguese`.
```
||videoszoofilia.net/wp-content/uploads/*/banner$image
```